### DISCUSSION TITLE
Different icon after keyframe interpolation

### DISCUSSION THREAD

**Discussion by sasislm2** [keiwando/toonsquid#180] [Ideas]
Need some visual difference between normal and eased keyframes.
My feature requests:
1. Need some basic layer effects like motion blur, glow, drop shadow, etc.
2. Options to hide layer thumbnails in the timeline (it takes more space)
3. Make animation properties a drop-down list (unused properties take up more space).
4. Custom graph editors

**Comment by DinoAwakening** [comment]
You just had some of the best ideas for the app! Also; we need Lens Blur, and a filter parameter for Hue, Saturation and Brightness. :D
